ambry-messageformat/src/main/java/com/github/ambry/messageformat/MessageFormatRecord.java

```diff
@@ -1756,6 +1754,78 @@ public static BlobData deserializeBlobRecord(CrcInputStream crcStream) throws IO
     }
   }
 
+  /**
+   *  - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   * |         |           |              |            |            |            |
+   * | version | blobType  | isCompressed |    size    |  content   |     Crc    |
+   * |(2 bytes)| (2 bytes) | (1 byte)     |  (8 bytes) |  (n bytes) |  (8 bytes) |
+   * |         |           |              |            |            |            |
+   *  - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   *  version    - The version of the blob record
+   *
+   *  blobType   - The type of the blob
+   *
+   *  isCompressed - Whether the content is compressed.
+   *
+   *  size       - The size of the blob content
+   *
+   *  content    - The actual content that represents the blob
+   *
+   *  crc        - The crc of the blob record
+   *
```

Comment:
For better readability, can you specify what CRC stands for here?

Reply:
CRC is Cyclic Redundancy Code.  It is better known by CRC than by the name, just like JPEG.